src/ol_infrastructure/infrastructure/aws/s3_sites/__main__.py

```diff
@@ -26,3 +27,28 @@
 export('xpro_certs_bucket', xpro_legacy_certs_site.site_bucket.bucket)
 export('xpro_certs_distribution_id', xpro_legacy_certs_site.cloudfront_distribution.id)
 export('xpro_certs_acm_cname', xpro_legacy_certs_site.site_tls.domain_name)
+
+# Static site for hosting beta releases of OCW content built with Hugo course publisher
+mitodl_zone_id = dns_stack.get_output('odl_zone_id')
+ocw_beta_site_config = S3ServerlessSiteConfig(
+    site_name='ocw-beta-course-site',
+    domains=['ocw-beta.odl.mit.edu'],
+    bucket_name='ocw-beta-course-site',
+    tags={'OU': 'open-courseware', 'Environment': 'operations'}
+)
+ocw_beta_site = S3ServerlessSite(xpro_legacy_certs_site_config)
```

Comment:
what does `xpro` have to do with ocw_beta?

Reply:
Nothing at all, except copy :spaghetti: